testsuite/tests/typing-unboxed/test.ml

```diff
@@ -631,7 +631,8 @@ external g : (float [@untagged]) -> float = "g" "g_nat";;
 Line 1, characters 14-19:
 1 | external g : (float [@untagged]) -> float = "g" "g_nat";;
                   ^^^^^
-Error: Don't know how to untag this type. Only "int" can be untagged.
+Error: Don't know how to untag this type.
+       Only "int", "char", "bool" and other immediate types can be untagged.
```

Comment:
Initially I liked this error message but, now that I have seen it a handful of times, I find it too long. I propose `Only "int" and other immediate types can be untagged.`

Also: visually I'm not fond of the line break between the two sentences -- I don't like that the first line is shorter than the second line. Instead of

```
Error: Don't know how to untag this type.
       Only "int" and other immediate types can be untagged.
```

I propose to format it as such:

```
Error: Don't know how to untag this type. Only "int"
       and other immediate types can be untagged.
```

Reply:
Done.